pkg/cmd/push/cmd.go

```diff
@@ -33,22 +33,26 @@ import (
 
 const (
 	shortDesc = `Upload a modelkit to a specified registry`
-	longDesc  = `This command pushes modelkits to a remote registry.
+	longDesc  = `This command pushes modelkits from local storage to a remote registry.
 
-The modelkits should be tagged with the target registry and repository before
-they can be pushed`
+If specified without a destination, the ModelKit should be tagged locally before
+pushing.`
```

Comment:
I think it "must" be tagged otherwise push will fail.

Reply:
Updated, but I think the wording here is generally strange -- if you type `kit push quay.io/myorg/mymodel:latest`, what does that mean if you don't have a modelkit tagged `quay.io/myorg/mymodel:latest`?